src/Helper/PaymentHelper.php

```diff
@@ -128,8 +100,8 @@ private function getPaymentSettings() {
    * @return string
    *   The Checkout key.
    */
-  public function getCheckoutKey() {
-    return $this->getPaymentSettings()['checkout_key'];
+  public function getCheckoutKey(): string {
```

Comment:
This function can return `NULL`:

```suggestion
  public function getCheckoutKey(): ?string {
```